src/octree/mod.rs

```diff
@@ -77,16 +77,29 @@ impl<L, I> Octree<L, I> {
         if !node.span().contains(p) {
             return None;
         }
+        let mut depth = 0;
         loop {
             if node.is_leaf() {
                 return Some(node);
             } else {
-                node = node
+                let parent_span = node.span();
+                let mut children_spans = Vec::new();
+                node = match node
                     .into_children()
                     .unwrap()
                     .into_iter()
-                    .find(|c| c.span().contains(p))
-                    .unwrap();
+                    .inspect(|c| children_spans.push(c.span()))
+                    .find(|c| c.span().contains(p)) {
+                    Some(c) => c,
+                    None => {
+                        println!("{:?}", p);
+                        println!("parent span: {:?}", parent_span);
+                        print!("depth: {}", depth);
+                        println!("cspans: {:#?}", children_spans);
+                        panic!();
+                    }
+                };
+                depth += 1;
```

Comment:
I think all those changes should be reverted. They were only for debugging purposes, right? I *think* we won't need to debug that particular section anymore.